java/fury-core/src/main/java/org/apache/fury/meta/MetaStringEncoder.java

```diff
@@ -57,7 +59,10 @@ public MetaString encode(String input, Encoding[] encodings) {
     if (input.isEmpty()) {
       return new MetaString(input, Encoding.UTF_8, specialChar1, specialChar2, new byte[0]);
     }
-    if (!StringSerializer.isLatin(input.toCharArray())) {
+    if (!StringUtils.isLatin(
+        input.toCharArray(),
+        Platform.CHAR_ARRAY_OFFSET,
+        StringSerializer.MULTI_CHARS_NON_LATIN_MASK)) {
```

Comment:
`MULTI_CHARS_NON_LATIN_MASK` should be moved to StringUtils too

Reply:
Moved the `MULTI_CHARS_NON_LATIN_MASK` to `StringUtils` as a static final field. Code looks cleaner, thanks for the feedback!